src/excels/sorttingfilters/sorts/SortByDate.tsx

```diff
@@ -1,33 +1,44 @@
 import React from 'react';
 import {Text, View} from 'react-native';
 import RNPickerSelect from 'react-native-picker-select';
-import {arrayofsort, TPrObject} from '../../../constant/constants';
+import {TPrObject} from '../../../interfaces/interfaces';
 import orderBy from 'lodash/orderBy';
 import getGlobalObjectStore from '../../../stores/GlobalObjectStore';
 import {observer} from 'mobx-react';
 import getSortFilterStore from '../../../stores/SortFilterStore';
 import getExcelStore from '../../../stores/ExcelStore';
+import getLanguageStore from '../../../stores/LanguageStore';
 
 const SortByDate = observer(() => {
-  const itemarray = arrayofsort.map(item => {
+  const itemarray = getLanguageStore.get('arrayofsort').map((item: string) => {
     return {label: item, value: item};
   });
 
   const setDateSort = (event: React.ChangeEvent) => {
-    const datesorting = event.toString() === 'desc' ? 'asc' : 'desc';
-    const newobjectarray = orderBy(
-      getGlobalObjectStore().arrayofobjects.get(),
-      (obj: TPrObject) => obj.Mydate,
-      [datesorting],
-    )
+    let newobjectarray;
+    if (event.toString() === 'تنازلي' || event.toString() === 'desc') {
```

Comment:
you can have one check using getLanguageStore.get no?

Reply:
noted, yes and done